atomic_reactor/odcs_util.py

```diff
@@ -81,6 +86,10 @@ def start_compose(self, source_type, source, packages=None, sigkeys=None, arches
         if arches is not None:
             body['arches'] = arches
 
+        if multilib_arches:
+            body['multilib_arches'] = multilib_arches
+            body['multilib_method'] = multilib_method
```

Comment:
Because the default value is `None` for this method, we might want to do something like this:
```python
body['multilib_method'] = multilib_method or []
```

I think `None`, which gets converted to json's `null` is not a valid value.

Reply:
Ah good catch! So maybe we need to handle default value here instead of `resolve_composes` plugin.

Reply:
Fixed.